src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -3582,6 +3626,24 @@ private Response verifyPut(String url, String body, String expectedContentType,
             .response();
   }
 
+
+  private void verifyLocationQuantity(Location location, CompositePoLine.OrderFormat orderFormat) {
+    switch (orderFormat) {
+      case P_E_MIX:
+        assertEquals(location.getQuantityPhysical() + location.getQuantityElectronic(), location.getQuantity().intValue());
```

Comment:
It would be nice if we were consistent wrt using `assertThat(..., equalT(...))` and `assertEquals(...)`